.github/workflows/ci.yml

```diff
@@ -51,6 +51,21 @@ jobs:
           # this will also kill the parent container
           sudo killall -9 creditcoin-node
 
+      - name: Start docker-compose
+        run: |
+          docker-compose up -d
+
+      - name: Sanity check for persistently mounted directory
+        run: |
+          sleep 60
+          docker exec creditcoin-validator ls -ld /creditcoin-node/data
+          docker exec creditcoin-validator ls -la /creditcoin-node/data
```

Comment:
Is there a useful way to assert this output in the ci?

Reply:
>  Is there a useful way to assert this output in the ci?

What do you mean exactly? I'm not sure I follow.

Meanwhile I've removed the "[Make sure /creditcoin-node/data is present inside the container](https://github.com/gluwa/creditcoin/pull/1145/commits/c1357d794cf52ced0daeb6689a0e7a51b54a7f4c)" commit and retried this CI job to make sure it will fail w/o the fix:
https://github.com/gluwa/creditcoin/actions/runs/5323529135/jobs/9641539867

Reply:
I guess I meant that I wanted to know if we can check that it's actually being populated properly.

Reply:
Failing CI job without the fix: https://github.com/gluwa/creditcoin/actions/runs/5323529135/jobs/9641539867 

the result is that once the container starts it will try to write to `/creditcoin-node/data` which is owned by root (when no fix applied) and the container itself will crash. Subsequently the `ls` commands will not work because there's no container to execute these commands on.